Update SCIM docs to include section on SA auth (#1249)
wandb/docs: content/guides/hosting/iam/scim.md

## Authentication

Access to the SCIM API can be authenticated in two ways:

> ### 1. User Authentication

Reviewer: ```suggestion
### Users
```

An organization or instance admin can use basic authentication with their API key to access the SCIM API. Set the HTTP request's `Authorization` header to the string `Basic` followed by a space, then the base-64 encoded string in the format `username:API-KEY`. In other words, replace the username and API key with your values separated with a `:` character, then base-64-encode the result. For example, to authorize as `demo:p@55w0rd`, the header should be `Authorization: Basic ZGVtbzpwQDU1dzByZA==`.

> ### 2. Service Account Authentication

Reviewer: ```suggestion
### Service accounts
```

> Organization service accounts can also access the SCIM API. For service account authentication, the username is left blank and only the API key is used. The API key for service accounts can be found in the service account tab in the organization dashboard.

Reviewer: ```suggestion
An organization service account with the `admin` role can access the SCIM API. The username is left blank and only the API key is used. Find the API key for service accounts in the **Service account** tab in the organization dashboard. Refer to [Organization-scoped service accounts]({{< relref "/guides/hosting/iam/service-accounts.md/#organization-scoped-service-accounts" >}}).
```

> Set the HTTP request's `Authorization` header to the string `Basic` followed by a space, then the base-64 encoded string in the format `:API-KEY` (notice the colon at the beginning with no username). For example, to authorize with only an API key such as `sa-p@55w0rd`, the header should be `Authorization: Basic OnNhLXBANTV3MHJk`.

Reviewer: ```suggestion
Set the HTTP request's `Authorization` header to the string `Basic` followed by a space, then the base-64 encoded string in the format `:API-KEY` (notice the colon at the beginning with no username). For example, to authorize with only an API key such as `sa-p@55w0rd`, set the header to: `Authorization: Basic OnNhLXBANTV3MHJk`.
```
